packages/client/lib/index.js

```diff
@@ -33,19 +32,28 @@ module.exports = class ApiClient {
     // Connect to each peer to get an updated list of peers until a success response
     const client = new ApiClient('http://', version)
     client.http.setTimeout(5000)
+
+    let peers = []
     for (const peer of networkPeers) {
       const peerUrl = `http://${peer.ip}:${peer.port}`
 
       // This method should not crash when a peer fails
       try {
         client.http.host = peerUrl
+        // On V2, this endpoint returns the port used by `core-p2p``
         const { data } = await client.resource('peers').all()
 
         let responsePeers = []
         if (version === 1 && data.success && data.peers) {
           responsePeers = data.peers
         } else if (version === 2 && data.data.length) {
-          responsePeers = data.data
+          // Use the default port used by `core-api`
+          responsePeers = data.data.map(peer => {
+            return {
+              ...peer,
+              port: 4003
```

Comment:
that should be taken from config instead of hardcoded

Reply:
I could change that a little, but the API client can be used on a browser, so it shouldn't depend on other packages to be lighter.